tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -82,6 +82,7 @@ def task(event):
         level = "event['input']"
         grans = []
         for ev_granule in event["input"]["granules"]:
+            recovery_bucket_override = ev_granule.get("recoverybucketoverride", None)
```

Comment:
recommend creating a global variable `INPUT_RECOVERY_BUCKET_OVERRIDE_KEY = "recoveryBucketOverride"` and then changing this line to `recovery_bucket_override = ev_granule.get(INPUT_RECOVERY_BUCKET_OVERRIDE_KEY, None)` to match current coding styles.

Reply:
added.